qiita_db/metadata_template/test/test_sample_template.py

```diff
@@ -1011,7 +1011,7 @@ def test_create(self):
                 'taxon_id': '9606',
                 'scientific_name': 'homo sapiens'}}
         for s_id in exp_sample_ids:
-            self.assertEqual(st[s_id]._to_dict(), exp_dict[s_id])
+            self.assertDictEqual(st[s_id]._to_dict(), exp_dict[s_id])
```

Comment:
In part, this change is because JSONB dicts are unordered, right?

Reply:
That's correct!